student_class.rb

```diff
@@ -1,12 +1,17 @@
-require './Person'
-
+require './person_class'
+require './classroom'
 class Student < Person
   def initialize(age, _name, _parent_permission, classroom)
-    super(age, name: 'Unknown', _parent_permission: true)
+    super(age, _name = 'Unknown', _parent_permission = true)
     @classroom = classroom
   end
 
   def play_hooky
     '¯\(ツ)/¯'
   end
+
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
+  end
```

Comment:
Great job on implementing the `has-many` / `belongs_to` relationship between student and classroom :+1: